src/reynir_correct/checker.py

```diff
@@ -186,17 +186,18 @@ class GreynirCorrect(Greynir):
     _lock = Lock()
 
     def __init__(self, **options: Any) -> None:
-        super().__init__()
+        super().__init__(**options)
         self._annotate_unparsed_sentences = options.pop(
             "annotate_unparsed_sentences", True
         )
-        if options:
-            raise ValueError(f"Unknown option(s) for GreynirCorrect: {options}")
+        self._options = options
```

Comment:
This is a bit dodgy. Is this only for options that will potentially be passed to tokenize_and_correct()?

Reply:
Yes. Another option, as we have defined the possible options, is to add them to a set, check here if any in options are not in that set, and then raise a ValueError.